source/ranges.tex

```diff
@@ -9341,6 +9357,550 @@
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to:
+\begin{codeblock}
+--@\exposid{current_}@;
+--@\exposid{pos_}@;
+return *this;
+\end{codeblock}
+\end{itemdescr}
+
+\indexlibrarymember{operator--}{enumerate_view::\exposid{iterator}}%
+\begin{itemdecl}
+constexpr @\exposid{iterator}@ operator--(int) requires @\libconcept{bidirectional_range}@<@\exposid{Base}@>;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to:
+\begin{codeblock}
+auto temp = *this;
+--*this;
+return temp;
+\end{codeblock}
+\end{itemdescr}
+
+\indexlibrarymember{operator+=}{enumerate_view::\exposid{iterator}}%
+\begin{itemdecl}
+constexpr @\exposid{iterator}@& operator+=(difference_type n)
+  requires @\libconcept{random_access_range}@<@\exposid{Base}@>;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to:
+\begin{codeblock}
+@\exposid{current_}@ += n;
+@\exposid{pos_}@ += n;
+return *this;
+\end{codeblock}
+\end{itemdescr}
+
+\indexlibrarymember{operator-=}{enumerate_view::\exposid{iterator}}%
+\begin{itemdecl}
+constexpr @\exposid{iterator}@& operator-=(difference_type n)
+  requires @\libconcept{random_access_range}@<@\exposid{Base}@>;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to:
+\begin{codeblock}
+@\exposid{current_}@ -= n;
+@\exposid{pos_}@ -= n;
+return *this;
+\end{codeblock}
+\end{itemdescr}
+
+\indexlibrarymember{operator==}{enumerate_view::\exposid{iterator}}%
+\begin{itemdecl}
+friend constexpr bool operator==(const @\exposid{iterator}@& x, const @\exposid{iterator}@& y) noexcept;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to: \tcode{return x.\exposid{pos_} == y.\exposid{pos_};}
+\end{itemdescr}
+
+\indexlibrarymember{operator<=>}{enumerate_view::\exposid{iterator}}%
+\begin{itemdecl}
+friend constexpr strong_ordering operator<=>(const @\exposid{iterator}@& x, const @\exposid{iterator}@& y) noexcept;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to: \tcode{return x.\exposid{pos_} <=> y.\exposid{pos_};}
+\end{itemdescr}
+
+\indexlibrarymember{operator+}{enumerate_view::\exposid{iterator}}%
+\begin{itemdecl}
+friend constexpr @\exposid{iterator}@ operator+(const @\exposid{iterator}@& x, difference_type y)
+  requires @\libconcept{random_access_range}@<@\exposid{Base}@>;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to:
+\begin{codeblock}
+auto temp = x;
+temp += y;
+return temp;
+\end{codeblock}
+\end{itemdescr}
+
+\indexlibrarymember{operator+}{enumerate_view::\exposid{iterator}}%
+\begin{itemdecl}
+friend constexpr @\exposid{iterator}@ operator+(difference_type x, const @\exposid{iterator}@& y)
+  requires @\libconcept{random_access_range}@<@\exposid{Base}@>;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to: \tcode{return y + x;}
+\end{itemdescr}
+
+\indexlibrarymember{operator-}{enumerate_view::\exposid{iterator}}%
+\begin{itemdecl}
+constexpr @\exposid{iterator}@ operator-(const @\exposid{iterator}@& x, difference_type y)
```

Comment:
Most definitely. It's present in the class synopsis, and an error without it (because there'd be 3 parameters to the binary operator, with the implicit `this`).